DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UITabBarRecorder.swift

```diff
@@ -76,3 +128,20 @@ internal struct UITabBarWireframesBuilder: NodeWireframesBuilder {
     }
 }
 #endif
+
+fileprivate extension UIImage {
+    var uniqueDescription: String? {
```

Comment:
suggestion/ We may want to add API comment with some of the contexts added in implementation. Notably, from the caller stand-point it might be interesting:
- that it is calculated from `CGImage` props,
- it prefers performance over acurracy (collisions are possible),
- when it may return `nil`.

Reply:
Good idea, I've updated ✅